src/sessions/video-session-handler.ts

```diff
@@ -290,19 +294,39 @@ export default class VideoSessionHandler extends BaseSessionHandler {
     this.setupTransceivers(session);
 
     const attachParams = { audioElement, videoElement };
-    if (session.tracks.length) {
-      session.tracks.forEach((track) => {
-        this.log(LogLevels.info, 'Incoming track', { track, conversationId: session.conversationId });
+
+    const handleIncomingTracks = (session: IJingleSession, tracks: MediaStreamTrack | MediaStreamTrack[]) => {
```

Comment:
This logic was duplicated below in the `if/else` block. Just moved it here and added the `setSinkId` call to it.